src/mappings/DaoCreator.ts

```diff
@@ -0,0 +1,12 @@
+import 'allocator/arena'
+export { allocate_memory }
+
+import { Entity, Value, store, crypto, ByteArray } from '@graphprotocol/graph-ts'
+
+import { NewOrg } from '../types/DaoCreator/DaoCreator'
+
+export function handleNewOrg(event: NewOrg): void {
```

Comment:
how about adding a function to get dao members here ? 
by looking on the controller mint reputations events